tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -197,7 +200,8 @@ async Task PerformMultiPhasesProbeTest()
         async Task RunPhase(SnapshotProbe[] snapshotProbes, ProbeAttributeBase[] probeData, bool isMultiPhase = false, int phaseNumber = 1)
         {
             var definition = DebuggerTestHelper.CreateProbeDefinition(snapshotProbes);
-            SetProbeConfiguration(definition);
+            SetRcmConfiguration(definition, LiveDebugger.Product.Name, EnvironmentHelper.SampleName);
```

Comment:
Nitpick - since this pattern repeats itself several times (and will also repeat in the future), maybe it'd be good to extract out an overload of `SetRcmConfiguration` that just received the `snapshotProbes` (and calls the base implementation of `SetRcmConfiguration` )?

Reply:
Refactored a bit at f1c42de650ba4a33e2b6ca10eb785b54bded2f08

Reply:
Thanks!